<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>669</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1078</width>
    <height>669</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1078</width>
    <height>669</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SimCell</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1081</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>DCMSS</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_1">
         <property name="text">
          <string>Salveaza imaginea</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QScrollBar" name="horizontalScrollBarDcmss">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>761</width>
        <height>17</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBarDcmss">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>60</y>
        <width>20</width>
        <height>431</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plotDcmss" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-1</y>
        <width>1081</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_1</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>horizontalScrollBarDcmss</zorder>
     <zorder>verticalScrollBarDcmss</zorder>
     <zorder>plotDcmss</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Contois-Fujimoto </string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Salveaza imaginea</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QScrollBar" name="horizontalScrollBarContois">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>761</width>
        <height>17</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plotContois" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBarContois">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>60</y>
        <width>20</width>
        <height>431</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-1</y>
        <width>1081</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>horizontalScrollBarContois</zorder>
     <zorder>plotContois</zorder>
     <zorder>verticalScrollBarContois</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Olsson</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Salveaza imaginea</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QScrollBar" name="horizontalScrollBarOlsson">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>761</width>
        <height>17</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plotOlsson" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBarOlsson">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>60</y>
        <width>20</width>
        <height>431</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-1</y>
        <width>1081</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>layoutWidget_3</zorder>
     <zorder>horizontalScrollBarOlsson</zorder>
     <zorder>plotOlsson</zorder>
     <zorder>verticalScrollBarOlsson</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Bley</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Salveaza imaginea</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_5"/>
       </item>
      </layout>
     </widget>
     <widget class="QScrollBar" name="horizontalScrollBarBley">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>751</width>
        <height>17</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plotBley" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBarBley">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>60</y>
        <width>20</width>
        <height>431</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-1</y>
        <width>1081</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>layoutWidget_5</zorder>
     <zorder>horizontalScrollBarBley</zorder>
     <zorder>plotBley</zorder>
     <zorder>verticalScrollBarBley</zorder>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Teissier </string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Salveaza imaginea</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollBar" name="horizontalScrollBarTeissier">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>751</width>
        <height>17</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plotTeissier" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>-50</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBarTeissier">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>60</y>
        <width>20</width>
        <height>431</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-1</y>
        <width>1081</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>layoutWidget_4</zorder>
     <zorder>horizontalScrollBarTeissier</zorder>
     <zorder>plotTeissier</zorder>
     <zorder>verticalScrollBarTeissier</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuData">
     <property name="title">
      <string>Data</string>
     </property>
     <addaction name="actionInitial_data"/>
     <addaction name="actionAverage_data"/>
    </widget>
    <addaction name="menuData"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionInitial_data">
   <property name="text">
    <string>Initial data</string>
   </property>
  </action>
  <action name="actionAverage_data">
   <property name="text">
    <string>Average data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
